File: app/src/main/java/com/example/newyorktimearticles/data/model/MetaData.java
package com.example.newyorktimearticles.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class MetaData implements Serializable {
    @SerializedName("url")
    @Expose
    private String imgUrl;

    public String getImgUrl() {
        return imgUrl;
    }
}

File: app/src/main/java/com/gemidroid/newyorktimearticles/data/model/MetaData.kt
package com.gemidroid.newyorktimearticles.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class MetaData : Serializable {
    @SerializedName("url")
    @Expose
    val imgUrl: String? = null
}